### DISCUSSION TITLE
Exception while scanning for Crazyflie USB

### DISCUSSION THREAD

**Discussion by nbelles** [bitcraze/discussions#339] [Q&A]
Hello,

I'm currently trying to use the Crazyradio PA to connect to the Crazyflie 2.1. I have been able to connect to it many times before but just the other day when I went to run some of my scripts to connect to the Crazyflie, I get the error:
```
Exception while scanning for Crazyflie USB: 'NoneType' object has no attribute 'bcdDevice'
```
I then went back to the basics to make sure my code wasn't what was working incorrectly and tried running one of the examples. When running 
`crazyflie-lib-python/examples/radio/scan.py/` I get the full output:
```
Scanning interfaces for Crazyflies...
Exception while scanning for Crazyflie USB: 'NoneType' object has no attribute 'bcdDevice'
Crazyflies found:
```
Any thoughts? Thanks!

**Comment by nbelles** [comment]
Update: I switched computers and am not getting this error on a different computer. Still not sure why I'm getting this error on this one computer but I posted a new discussion about the errors I'm encountering on the new computer.

**Reply by ataffanel** [reply]
I am not sure of what could have happened but I have 2 hypothesis: either the pyusb package has been updated/downgraded by another install on the system or somehow the udev rule that allows you to access the usb Crazyradio is not active anymore and so you do not have access right to the radio.

**Reply by nbelles** [reply]
Thanks for the reply. I went back to it to try it one more time today and it just worked™ so I don't know what happened before. I'll follow up here again if I experience the issue again.
